buildenv/ansible/roles/openj9-build-environment/tasks/main.yml

```diff
@@ -22,20 +22,22 @@
 ---
 - name: Create a build folder
   file:
-    dest: "{{ build_dir }}" 
-    mode: 0755 
+    dest: "{{ build_dir }}"
+    mode: 0755
     state: directory
   tags:
     - buildenv
-- name: Get Dockerfile for the building environment
+- name: Get script to build docker image
   get_url:
-    dest: "{{ build_dir }}/Dockerfile-base"
-    url: "https://raw.githubusercontent.com/eclipse/openj9/master/buildenv/docker/{{ version }}/{{ platform }}/ubuntu16/Dockerfile"
-    mode: 0644
+    dest: "{{ build_dir }}/mkdocker.sh"
+    url: "https://raw.githubusercontent.com/eclipse/openj9/master/buildenv/docker/mkdocker.sh"
+    mode: 0755
     validate_certs: no
   tags:
     - buildenv
 - name: Build the docker image for the building environment
+  # docker/{{ version }}/{{ platform }}/ubuntu16
+  # bash mkdocker.sh --tag=openj9-{{ version }}-build --dist=ubuntu --version=16.04 --build
```

Comment:
Is this supposed to be a comment?

Reply:
No, it is a reminder that I need to learn enough about ansible to run that `bash` command.

Reply:
To merge in advance of fixing this problem, we should have an issue to address it later.

Reply:
It turned out to be easier to just fix it.